flocker/volume/filesystems/zfs.py

```diff
@@ -7,11 +7,13 @@
 from __future__ import absolute_import
 
 import os
+import libzfs_core
```

Comment:
We could perhaps have a zfs-requirements.txt and corresponding separate OS packages you need to install, so e.g. `clusterhq-flocker-zfs` if you want ZFS backend.